deps/rabbit/src/rabbit_stream_coordinator.erl

```diff
@@ -2013,12 +2014,26 @@ make_replica_conf(LeaderPid,
 
 make_writer_conf(Node, #stream{epoch = Epoch,
```

Comment:
This function is used inside the state machine which means we cannot reach outside the scope of the machine and into the environment as this is not deteministic.